src/components/Home/index.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable camelcase */
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import {
+  Container, Col, Row, Form, Card, Button,
+} from 'react-bootstrap';
+import ClipLoader from 'react-spinners/ClipLoader';
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import fetchCovidData from '../../redux/worldData/world.actions';
+import fetchCountryData from '../../redux/countryData/country.actions';
+import './index.css';
+import covid from '../covid.png';
+
+const override = css`
+  display: block;
+  margin: 0 auto;
+  border-color: red;
+`;
+
+const Home = () => {
```

Comment:
- [ ] Please take a look at the image below. As I have highlighted your Home page is missing the dual-alternating colors as we have in the design. Kindly add this styling.

![adewu-home-color](https://user-images.githubusercontent.com/36542986/142658368-5a016fb0-950c-453f-a9de-92076c5d21c3.jpg)